sycl/source/detail/platform_impl.hpp

```diff
@@ -76,11 +76,10 @@ class platform_impl {
 
   /// \return an instance of OpenCL cl_platform_id.
   cl_platform_id get() const {
-    if (is_host() || getPlugin().getBackend() != cl::sycl::backend::opencl) {
-      throw invalid_object_error(
-          "This instance of platform doesn't support OpenCL interoperability.",
-          PI_INVALID_PLATFORM);
-    }
+    if (is_host())
```

Comment:
why did you remove the BE check here?

Reply:
Because platform::get() method is used internally in creating of sycl::context and it comes to exception there.
But now, I found a solution and restore this check.